src/main/java/gg/solarmc/kitpvp/messaging/Prefix.java

```diff
@@ -1,23 +1,54 @@
 package gg.solarmc.kitpvp.messaging;
 
+import org.bukkit.Server;
 import org.bukkit.entity.Player;
 
+import java.awt.*;
+import java.util.UUID;
+
 public enum Prefix {
 
-    ACTIONBAR("[ACTIONBAR]", Player::sendActionBar),
-    CHAT("[CHAT]",Player::sendMessage),
-    TITLE("[TITLE]",(s,e) -> {
-        s.sendTitle(e,"",0,10,10);
+    ACTIONBAR("[ACTIONBAR]", (s,e,ser) -> {
+        Player player = ser.getPlayer(s);
+
+        if (player != null) {
+
+            //because we have kyori now i need to figure out how to change this
+        }
     }),
-    SUBTITLE("[SUBTITLE]", (s,e) -> {
-        s.sendTitle("",e,0,10,10);
+    CHAT("[CHAT]",(s,e,ser) -> {
+        Player player = ser.getPlayer(s);
+
+        if (player != null) {
```

Comment:
A lot of code is duplicated with these null checks. If you use Player as your parameter type you won't need them

Reply:
The reason i switched the code over to null checks was to solve the perceived issue of using player as the parameter type. What i am going to do is roll this back by a commit.